node/src/primitives.rs

```diff
@@ -9,6 +9,37 @@ use rand::prelude::IteratorRandom;
 use serde::{Deserialize, Serialize};
 use std::fmt::{Debug, Display};
 
+// There's an exact enum in near-crypto, but it doesn't implement required traits (eq, hash, etc.)
+#[derive(
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    PartialOrd,
+    Ord,
+    Hash,
+    BorshSerialize,
```

Comment:
Do we need these Borsh serde derives and standard serde derives if we're just using it as an internal type?